### PR TITLE
Adds a non-swimmer quirk (you drown in pools)

### PR DESCRIPTION
**Pull request by bruhlookatthisdood**
# About The Pull Request

Non-swimmer quirk when equipped will cause you to drown in the pool because you can't swim, you will also take more damage than a normal person while drowning. Also causes you to slip on puddles because you're not used to water (idk).

## Why It's Good For The Game

More content for the pool seems fun and another random quirk

## Testing Photographs and Procedure

image

## Changelog

:cl:
add: Non-swimmer quirk when equipped will cause you to drown in the pool because you can't swim, you will also take more damage than a normal person while drowning. Also causes you to slip on puddles because you're not used to water (idk).
/:cl:

### CODE REVIEW

File: code/modules/pool/components/swimming.dm [hunk 1]
```diff
@@ -106,7 +106,7 @@
 		if(bob_tick)
 			animate(L, time=5, pixel_y = 0)
 			bob_tick = 0
-	if(!floating && is_drowning(L))
+	if(!floating && is_drowning(L) || HAS_TRAIT(L, TRAIT_NONSWIMMER) && !floating)
```
Comment (Oricana-16) [inline code/modules/pool/components/swimming.dm:109]:
```suggestion
	if(!floating && (is_drowning(L) || HAS_TRAIT(L, TRAIT_NONSWIMMER)))
```

**Review by Oricana-16 — COMMENTED** [review]
I can't really check from mobile but it Feels like there should be traits in the monkestation folder. Check where stowaway/jailbird or any of the others were put just to make sure.

**Comment by KoboldCommando** [comment]
This is a great idea but it feels like a "free points" perk with how rarely the pool comes into play.  Until/unless we do a perk cost rebalance even 1 point should impact the game fairly frequently.

Maybe it could have some other liquid interaction, like if you always slipped on liquid puddles?

**Comment by dwasint** [comment]
this is a good idea but it should be a zero point or have more downside, pools are so uncommon that this is basically a free point. Extra slip chance in puddles and extra damage from drowning is liquids would be a good addition

**Comment by bruhlookatthisdood** [comment]
i actually belive that this quirk wasn't much of a good idea, its either 1 point which would be too much or 0 which would leave no reason to get the quirk. and the way i've balanced it made it not good to get.

**Pull request closed without merging**